tables/ru-litbrl.ctb

```diff
@@ -376,33 +438,82 @@ always \x00ab 236 (opening quotation mark)
+attribute upperlatin ABCDEFGHIJKLMNOPQRSTUVWXYZ\x00c0\x00c2\x00c4\x00c7\x00c8\x00c9\x00ca\x00ce\x00d4\x00d6\x00db\x00dc\x0152\x1e9e
+attribute lowerlatin abcdefghijklmnopqrstuvwxyz\x00df\x00e0\x00e2\x00e4\x00e7\x00e8\x00e9\x00ea\x00ee\x00f4\x00f6\x00fb\x00fc\x0153
+attribute uppercyrillic \x0401\x0410\x0411\x0412\x0413\x0414\x0415\x0416\x0417\x0418\x0419\x041a\x041b\x041c\x041d\x041e\x041f\x0420\x0421\x0422\x0423\x0424\x0425\x0426\x0427\x0428\x0429\x042a\x042b\x042c\x042d\x042e\x042f\x0404\x0405\x0406\x0407\x040b\x0460\x0462\x0464\x0466\x0468\x046a\x046c\x046e\x0470\x0472\x0474\x0476\x0478\x047a\x047e\x0480\xa650\xa652\xa656\xa64a
+attribute lowercyrillic \x0430\x0431\x0432\x0433\x0434\x0435\x0436\x0437\x0438\x0439\x043a\x043b\x043c\x043d\x043e\x043f\x0440\x0441\x0442\x0443\x0444\x0445\x0446\x0447\x0448\x0449\x044a\x044b\x044c\x044d\x044e\x044f\x0451\x0454\x0455\x0456\x0457\x045b\x0461\x0463\x0465\x0467\x0469\x046b\x046d\x046f\x0471\x0473\x0475\x0477\x0479\x047b\x047f\x0481\xa651\xa653\xa657\xa64b
 attribute uppergreek \x0391\x0392\x0393\x0394\x0395\x0396\x0397\x0398\x0399\x039a\x039b\x039c\x039d\x039e\x039f\x03a0\x03a1\x03a3\x03a4\x03a5\x03a6\x03a7\x03a8\x03a9
 attribute lowergreek \x03b1\x03b2\x03b3\x03b4\x03b5\x03b6\x03b7\x03b8\x03b9\x03ba\x03bb\x03bc\x03bd\x03be\x03bf\x03c0\x03c1\x03c3\x03c4\x03c5\x03c6\x03c7\x03c8\x03c9
 attribute fractions \x00bd\x2155\x00bc\x00be\x2150\x2151\x2152\x2153\x2154\x2156\x2157\x2158\x2159\x215a\x215b\x215c\x215d\x215e\x2189
 swapcd updigit 1234567890 1,12,14,145,15,124,1245,125,24,245
 
-# Adjust spacing around signs according to the Russian braille rules.
-noback context `["-\s"] @36
-noback context [$s]"-"$s ?
+# Adjust spacing around symbols according to the Russian braille rules.
+noback correct $D[","$s] "\x2820,"
+noback correct $d[","$s] "\x2820,"
+noback correct $D[";"$s] "\x2820;"
+noback correct $d[";"$s] "\x2820;"
+noback correct ","[$s] ?
+noback correct ";"[$s] ?
+noback correct ["*"$s"*"$s"*"] "***"
+noback correct `["-"$s] "-"
+noback correct `["\x2013"$s] "\x2013"
+noback correct `["\x2014"$s] "\x2014"
+noback correct `["\x2015"$s] "\x2015"
+noback correct [$s]"-"$s ?
+noback correct [$s]"\x2013"$s ?
+noback correct [$s]"\x2014"$s ?
+noback correct [$s]"\x2015"$s ?
+noback correct "="[$s] ?
+noback correct [$s]"\x00d7" ?
+noback correct "\x00d7"[$s] ?
+noback correct "\x2260"[$s] ?
+noback correct "\x2264"[$s] ?
+noback correct "\x2265"[$s] ?
+noback correct "\x2245"[$s] ?
+noback correct "\x2248"[$s] ?
+noback correct "~"[$s] ?
+noback correct $D[$s]"%" ?
+noback correct $d[$s]"%" ?
+noback correct "\x00a7"[$s] ?
+noback correct [$s]"°" ?
+noback correct "\x2044"[$s] ?
+noback correct "\x2236"[$s] ?
+noback correct "\x2227"[$s] ?
+noback correct "\x2228"[$s] ?
+noback correct "\x00ac"[$s] ?
```

Comment:
Looks good, but I wonder if we could simplify this further by defining a custom attribute, e.g.

    attribute dropspaceafter ,;-\x2013\x2014\x2015=\x00d7\x2260\x2264\x2265\x2245\x2248~\x00a7\x2044\x2236\x2227\x2228\x00ac

Reply:
> Looks good, but I wonder if we could simplify this further by defining a custom attribute...

I didn't think about it, of course, it makes sense and I'll do it. But it's necessary two attributes because in some cases we ignore the space before a symbol, and sometimes after a symbol.